DigitalLearningSolutions.Web/Views/LearningPortal/Completed/Completed.cshtml

```diff
@@ -5,6 +5,7 @@
 
 <link rel="stylesheet" href="@Url.Content("~/css/shared/searchableElements.css")" asp-append-version="true">
 <link rel="stylesheet" href="@Url.Content("~/css/learningPortal/completed.css")" asp-append-version="true">
+<link rel="stylesheet" href="@Url.Content("~/css/learningPortal/courses.css")" asp-append-version="true">

```

Comment:
Could we not just import the courses.css in the completed.css with `@import "courses.scss"` to have the completed page css built in one stylesheet?

Reply:
Yes, and we probably could be importing the searchableElements in various places as well. Overall there's probably quite a lot of scss that is poorly set up in the project based on the amount of changes I was able to make when refactoring searchableElements.scss in 532.